project_role/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    project_role_ids = fields.One2many(
```

Comment:
I think this shouldn't be here. You already have roles in other menu entry.

Reply:
This is a purely usability enhancement, implemented with warning that roles are configured per-company

Reply:
The problem is this makes even slower the load of the settings, which is now very bloated loading everything for all the settings in any area. If you want to have this usability improvement, just put a link to the action as you can see for example for the pricelists when you select the option of multiple prices:

![seleccion_060](https://user-images.githubusercontent.com/7165771/49522523-23959580-f8a8-11e8-8ece-5850f0a7b98e.png)

Reply:
Understood, will adjust this. In UI it actually is like that:
![image](https://user-images.githubusercontent.com/345102/49523329-3614cc80-f8b2-11e8-8229-4ec99ff4d4fe.png)